Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/donut.yml

```diff
@@ -182,14 +182,14 @@
 # Tastes like donut, bitterness.
 
 - type: entity
-  name: blumpkin donut
+  name: bloompkin donut
   parent: FoodDonutBase
-  id: FoodDonutBlumpkin
-  description: Goes great with a mug of soothing drunken blumpkin.
+  id: FoodDonutBloompkin
+  description: Goes great with a mug of soothing drunken bloompkin.
```

Comment:
Descriptions for the bloompkin donuts kinda lost their ryhme gimmick making the dunken donuts joke not make much sense.

Reply:
Well i cant exactly make the joke work with this new name. Might as well just leave it